packages/expo/src/StoreReview.ts

```diff
@@ -1,21 +1,28 @@
 import { NativeModules, Platform, Linking } from 'react-native';
 import { Constants } from 'expo-constants';
 
+const {
+  ExponentStoreReview = {
```

Comment:
Split this out into the separate file, I know it's a bit verbose but it's good to set a consistent pattern across the modules that others can follow.